cmd/x-check-mailserver/flags.go

```diff
@@ -10,6 +10,7 @@ import (
 var (
 	fleet       = pflag.StringP("fleet", "f", params.FleetProd, "cluster fleet")
 	datadir     = pflag.StringP("datadir", "d", "/tmp", "home directory for node data")
+	privkey     = pflag.StringP("privkey", "p", "", "private key for connecting to nodes, ignores 0x")
```

Comment:
```suggestion
	privkey     = pflag.StringP("privkey", "p", "", "private key for connecting to nodes, hex encoded without any '0x' prefix")
```

Reply:
Except that's wrong, because I remove the prefix.